src/mp2_eri_gpw.F

```diff
@@ -271,6 +300,8 @@ SUBROUTINE mp2_eri_2c_integrate_gpw(qs_env, para_env_sub, dimen_RI, mo_coeff, my
                ALLOCATE (I_ab(nsgfa(iset), 1))
                I_ab = 0.0_dp
 
+               offset = offset_2d(iset, iatom)
```

Comment:
This assignment is not needed because the `offset` variable is use only once. Simply plug it in directly:

`MODULO(offset_2d(iset, iatom), para_env_sub%num_pe)`